spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc

```diff
@@ -780,13 +780,13 @@ This style of configuration works particularly well with the `SpringApplication`
 [source,yaml,indent=0,subs="verbatim"]
 ----
 	my:
-		service:
-			remote-address: 192.168.1.1
-			security:
-				username: admin
-				roles:
-				  - USER
-				  - ADMIN
+	  service:
+	    remote-address: "192.168.1.1"
```

Comment:
This PR is doing two different things here. Are the quote around the value necessary?

Reply:
Ha, I think the String type here is more suitable, so I just did it.